snapcraft/tests/fixture_setup.py

```diff
@@ -411,6 +413,119 @@ def call_effect(*args, **kwargs):
     return call_effect
 
 
+class FakeAptGetBuildDep(fixtures.Fixture):
+    '''Mock apt-get build-dep output'''
+
+    _PROLOG = dedent('''
+        NOTE: This is only a simulation!
+              apt-get needs root privileges for real execution.
+              Keep also in mind that locking is deactivated,
+              so don't depend on the relevance to the real current situation!
+              ''')
+    _READING = dedent('''
+        {}
+        Note, using file '{}' to get the build dependencies
+        Reading package lists...
+        Building dependency tree...
+        Reading state information...''')
+    _PROBLEMS = dedent('''
+        Some packages could not be installed. This may mean that you have
+        requested an impossible situation or if you are using the unstable
+        distribution that some required packages have not yet been created
+        or been moved out of Incoming.
+        The following information may help to resolve the situation:
+
+        The following packages have unmet dependencies:
+         builddeps:{}{} : {}
```

Comment:
I typically find that it's clearer to name these when used in a larger template like this. Otherwise changes become quite fragile.